src/OrchardCore.Build/Dependencies.props

```diff
@@ -39,6 +39,7 @@
     <PackageManagement Include="OpenIddict.AspNetCore" Version="3.1.1" />
     <PackageManagement Include="OpenIddict.Core" Version="3.1.1" />
     <PackageManagement Include="OrchardCore.Translations.All" Version="1.1.0" />
+    <PackageManagement Include="PdfPig" Version="0.1.5-alpha001" />
```

Comment:
It's no longer alpha https://www.nuget.org/packages/PdfPig/

Reply:
Thanks, updated here too.